apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -136,8 +137,11 @@ const getTypeAndNameByKey = (unitKey) => {
 }
 
 const getUnitNamesAndUnitTypes = (property, sectionKeys) => {
-    const selectedSections = get(property, ['map', 'sections'], []).filter(section => includes(sectionKeys, `${section.type}-${section.name}`))
-    const allSectionsUnits = getAllSectionsUnits(selectedSections)
+    const selectedSections = get(property, ['map', 'sections'], []).filter((section) => includes(sectionKeys, `${section.type}-${section.name}`))
+    const selectedParking = get(property, ['map', 'parking'], []).filter((section) => {
+        return includes(sectionKeys, `${PARKING_SECTION_TYPE}-${section.name}`)
```

Comment:
Why parking filter query is not consistent with section?

Reply:
Is there a possibility to make this code looks like that?
```suggestion
    const sectionTypeFilter = (section) => includes(sectionKeys, `${section.type}-${section.name}`)

    const selectedSections = get(property, ['map', 'sections'], []).filter(sectionTypeFilter)
    const selectedParking = get(property, ['map', 'parking'], []).filter(sectionTypeFilter)
```

Reply:
Because parking has section type equals `section`
<img width="343" alt="image" src="https://github.com/open-condo-software/condo/assets/25384290/d66a3c54-e206-4868-b4dc-0cbb9342e31b">
But `SectionNameInput` returns the parking section as "parking-n"
https://github.com/open-condo-software/condo/blob/435c91f0288151ac5c0feae790be35359fecef3b/apps/condo/domains/user/components/SectionNameInput.tsx#L62